setup.cfg

```diff
@@ -1,11 +1,5 @@
 [bdist_wheel]
 python_tag = py3
 
-[flake8]
-exclude = docs
-max-line-length = 88
-# W503 - line break before binary operator - black formats it that way
-ignore = W503
```

Comment:
Are these in a toml that was not uploaded?

Reply:
max-line length is default, ignore unneeded and the exclude unnecessary, because we explicitly pass the directories in our makefile